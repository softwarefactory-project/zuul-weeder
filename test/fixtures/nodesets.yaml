- nodeset:
    name: nodeset1
    nodes:
      - name: controller
        label: controller-label
      - name: compute1
        label: compute-label
      - name:
          - compute2
          - web
        label: compute-label
