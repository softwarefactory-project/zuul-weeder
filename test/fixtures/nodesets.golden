[ ZNodeset
    ( Nodeset
        { name = NodesetName "nodeset1"
        , labels =
            [ NodeLabelName "compute-label"
            , NodeLabelName "compute-label"
            , NodeLabelName "controller-label"
            ]
        }
    )
]